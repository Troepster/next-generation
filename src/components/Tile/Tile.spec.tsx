import React from 'react';
import renderer from 'react-test-renderer';
import { Tile } from './';

it('should render Tile and match snapshot', () => {
  const tree = renderer.create(<Tile alive={true} x={0} y={0} onClick={() => null} />).toJSON();
  expect(tree).toMatchSnapshot();
});
