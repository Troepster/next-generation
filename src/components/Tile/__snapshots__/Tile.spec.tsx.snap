// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Tile and match snapshot 1`] = `
<div
  className="sc-bdnylx hCphIg"
  onClick={[Function]}
/>
`;
